README.md

```diff
@@ -21,7 +21,6 @@ pip install becquerel
 
 * Reading additional `Spectrum` file types (N42, CHN, CSV)
 * Writing `Spectrum` objects to various standard formats
-* Fitting spectral features (e.g., gaussian lines with different background models)
```

Comment:
Do we want to note that Poisson likelihood fitting is still in the works?

Reply:
Sounds good